README.md

```diff
@@ -40,16 +40,93 @@ The Rucio file catalog is much flatter than that of a usual `POSIX` filesystems
 - Fix Mac OS X mouting issues
 - Surely a lot more...
 
-## How to build
```

Comment:
General: code kind like BASH must be lowercase to ensure compatibility.

Reply:
Okay. Noted!